projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -81,7 +82,12 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
 
   private subscription: Subscription | undefined = undefined;
 
-  constructor() {
+  public ngOnInit(): void {
+    this.formGroup = (new FormGroup(
+      this._getFormControls(),
+      this.getFormGroupControlOptions() as AbstractControlOptions,
+    ) as unknown) as TypedFormGroup<FormInterface>;
```

Comment:
Moving the creation of the `formGroup` from a property into the `ngOnInit` method is the big change here.

I cannot think of any use case where this would be an issue really but let me know if you do :+1: